### DISCUSSION TITLE
saya newbie

### DISCUSSION THREAD

**Discussion by riagie** [lintangtimur/ovoid#75] [General]
lanjutan dari 

#52 

untuk apliaksi yang di gunakan seperti Burp Suite, Charles Proxy, etc etc.
open aplikasi di mobile untuk cara dapat nya bagaimana ?
apakah run apk di pc ?

**Comment by lintangtimur** [comment]
Dulu saya melakukan dengan menjalankan apk di emulator seperti emulator dari Android Studio.
Selanjutnya dengan `burp suite` dan `objection`.

Untuk iOS belum pernah nyoba sniffing nya bagaimana

**Reply by riagie** [reply]
objection dari frida?

**Reply by lintangtimur** [reply]
Yes, https://github.com/sensepost/objection
